### ISSUE TITLE
French documentation is not available

### ISSUE THREAD

**Issue by xsellier** [godotengine/godot-docs#2961]
**Actual behavior**
- Open this URL with your favorite browser:
https://docs.godotengine.org/en/3.1/classes/class_object.html
- Switch to fr (or any other language)
- Documentation not found

**Expected behavior**
- Open this URL with your favorite browser:
https://docs.godotengine.org/en/3.1/classes/class_object.html
- Switch to fr (or any other language)
- Documentation opened in French

![broken-link-doc](https://user-images.githubusercontent.com/6616005/69998496-70946200-1524-11ea-9973-f5ba7b23fa9b.gif)

**Comment by FeatherAntennae** [comment]
Going in the documentation in french, the class reference isn't there at all even in the menu.

In fact I think only English has access to class reference and other versions than "latest".

**Comment by akien-mga** [comment]
Indeed, the localized documentation instances only exist for the `latest` branch, not `3.1`. And they don't mirror the class reference which is only in English and can't be translated for now (by choice).
